Migrate CSS for page-list component

Right now the CSS for this component lives in `styles.css`:
https://github.com/edgi-govdata-archiving/web-monitoring-ui/blob/master/src/css/styles.css#L130

This is vanilla CSS that is currently loaded at the global level but we are currently migrating our CSS to [CSS Modules](https://github.com/css-modules/css-modules).

Here is an [example PR](https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/393) where the `source-info` component was migrated over.

To accomplish this we need to do the following:

1. Add the CSS to the new pipeline by migrating it out of `styles.css` and into a new file called `page-list.css`. The directory structure should look like this:
```
src/components
└─┬ page-list
  ├── page-list.jsx
  └── page-list.css
```

2. In `page-list.jsx`, the new CSS file needs to imported:
```
import './page-list.css';
```

3. The `.page-list tbody tr` and any other selectors using more than one element/class name/etc., need to be refactored to be a single class name. Something like `.page-list-row`, for example, could work or something similar. These classes need to be added/replaced in the `page-list.jsx` file. Feel free to ask if you aren't sure where the new classes should go in the markup.

4. In `page-list.jsx`, the `className` tags needs to be added or replaced for `styleName` **for the component level styles only**. Global styles should stay in `className` tags and a component can have both `className` and `styleName` tags at the same time if necessary. Feel free to ask if you aren't sure if a style falls under either category. (This basically determines which classes get suffixed with a hash to make sure it is scoped at the component level).

5. Fix any tests that fail because you refactored the class names. You can run the tests using `npm test`.

Here is an example of `styleName` vs `className` being used in our codebase:
https://github.com/edgi-govdata-archiving/web-monitoring-ui/blob/06faddae40f19eb7785e87ffc2d33055500f2658/src/components/source-info/source-info.jsx#L65-L73

Hey @Mr0grog ,
So I am finished with fixing this issue but cannot seem to pre commit it due to an error with patterns not matching.
The files waiting for commit are CSS and JSX files.
I get this error : 
`> eslint --ignore-path .gitignore './**/*.{js,jsx}'


Oops! Something went wrong! :(

ESLint: 6.3.0.

No files matching the pattern "'./**/*.{js,jsx}'" were found.
Please check for typing mistakes in the pattern.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! web-monitoring-ui@0.1.0 lint: `eslint --ignore-path .gitignore './**/*.{js,jsx}'`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the web-monitoring-ui@0.1.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jaaro\AppData\Roaming\npm-cache\_logs\2019-10-01T03_46_08_840Z-debug.log`

Thanks.

--- a/src/components/__tests__/page-list.test.jsx
+++ b/src/components/__tests__/page-list.test.jsx
@@ -27,7 +27,7 @@ describe('page-list', () => {
         pages={simplePages}
       />
     );
-    expect(pageList.find('.page-list tbody tr').first().childAt(1).text())
+    expect(pageList.find('tbody tr').first().childAt(1).text())
       .toBe('NOAA - ncei.noaa.gov, EPA - www3.epa.gov');
   });
 });

--- a/src/components/__tests__/page-list.test.jsx
+++ b/src/components/__tests__/page-list.test.jsx
@@ -1,6 +1,6 @@
 /* eslint-env jest */
 
-import PageList from '../page-list';
+import PageList from '../page-list/page-list';
 import React from 'react';
 import {shallow} from 'enzyme';
 import simplePages from '../../__mocks__/simple-pages.json';

--- a/src/components/page-list/page-list.css
+++ b/src/components/page-list/page-list.css
@@ -1,35 +1,33 @@
-
 /* ===== List View ===== */
 .page-list {
-    table-layout: fixed;
-  }
-  
-  .page-list-tbody-tr {
-    cursor: pointer;
-  }
-  
-  .page-list-tbody-tr:hover {
-    background-color: #eee;
-  }
-  
-  .page-list-td {
-    overflow-wrap: break-word;
-    word-wrap: break-word;
-  }
-  
-  .page-list-th[data-name="capture-date"] {
-    width: 12%;
-  }
-  
-  .page-list-th[data-name="site"] {
-    width: 15%;
-  }
-  
-  .page-list-th[data-name="page-name"] {
-    width: 33%;
-  }
-  
-  .page-list-th[data-name="url"] {
-    width: 40%;
-  }
-  
+  table-layout: fixed;
+}
+
+.page-list-tbody-tr {
+  cursor: pointer;
+}
+
+.page-list-tbody-tr:hover {
+  background-color: #eee;
+}
+
+.page-list-td {
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+}
+
+.page-list-th[data-name="capture-date"] {
+  width: 12%;
+}
+
+.page-list-th[data-name="site"] {
+  width: 15%;
+}
+
+.page-list-th[data-name="page-name"] {
+  width: 33%;
+}
+
+.page-list-th[data-name="url"] {
+  width: 40%;
+}

--- a/src/components/page-list/page-list.css
+++ b/src/components/page-list/page-list.css
@@ -1,0 +1,35 @@
+
+/* ===== List View ===== */
+.page-list {
+    table-layout: fixed;
+  }
+  
+  .page-list-tbody-tr {
+    cursor: pointer;
+  }
+  
+  .page-list-tbody-tr:hover {
+    background-color: #eee;
+  }
+  
+  .page-list-td {
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+  }
+  
+  .page-list-th[data-name="capture-date"] {
+    width: 12%;
+  }
+  
+  .page-list-th[data-name="site"] {
+    width: 15%;
+  }
+  
+  .page-list-th[data-name="page-name"] {
+    width: 33%;
+  }
+  
+  .page-list-th[data-name="url"] {
+    width: 40%;
+  }
+  

--- a/src/components/page-list/page-list.css
+++ b/src/components/page-list/page-list.css
@@ -3,31 +3,31 @@
   table-layout: fixed;
 }
 
-.page-list-tbody-tr {
+.page-list>tbody>tr {
   cursor: pointer;
 }
 
-.page-list-tbody-tr:hover {
+.page-list>tbody>tr:hover {
   background-color: #eee;
 }
 
-.page-list-td {
+.page-list>td {
   overflow-wrap: break-word;
   word-wrap: break-word;
 }
 
-.page-list-th[data-name="capture-date"] {
+.page-list>th[data-name="capture-date"] {
   width: 12%;
 }
 
-.page-list-th[data-name="site"] {
+.page-list>th[data-name="site"] {
   width: 15%;
 }
 
-.page-list-th[data-name="page-name"] {
+.page-list>th[data-name="page-name"] {
   width: 33%;
 }
 
-.page-list-th[data-name="url"] {
+.page-list>th[data-name="url"] {
   width: 40%;
 }

--- a/src/components/page-list/page-list.css
+++ b/src/components/page-list/page-list.css
@@ -10,23 +10,23 @@
   background-color: #eee;
 }
 
-.page-list > td {
+.page-list > tbody > tr > td {
   overflow-wrap: break-word;
   word-wrap: break-word;
 }
 
-.page-list > th[data-name="capture-date"] {
+.page-list > tbody > tr > th[data-name="capture-date"] {
   width: 12%;
 }
 
-.page-list > th[data-name="site"] {
+.page-list > tbody > tr > th[data-name="site"] {
   width: 15%;
 }
 
-.page-list > th[data-name="page-name"] {
+.page-list > tbody > tr > th[data-name="page-name"] {
   width: 33%;
 }
 
-.page-list > th[data-name="url"] {
+.page-list > tbody > tr > th[data-name="url"] {
   width: 40%;
 }

--- a/src/components/page-list/page-list.css
+++ b/src/components/page-list/page-list.css
@@ -1,33 +1,32 @@
-/* ===== List View ===== */
 .page-list {
   table-layout: fixed;
 }
 
-.page-list>tbody>tr {
+.page-list > tbody > tr {
   cursor: pointer;
 }
 
-.page-list>tbody>tr:hover {
+.page-list > tbody > tr:hover {
   background-color: #eee;
 }
 
-.page-list>td {
+.page-list > td {
   overflow-wrap: break-word;
   word-wrap: break-word;
 }
 
-.page-list>th[data-name="capture-date"] {
+.page-list > th[data-name="capture-date"] {
   width: 12%;
 }
 
-.page-list>th[data-name="site"] {
+.page-list > th[data-name="site"] {
   width: 15%;
 }
 
-.page-list>th[data-name="page-name"] {
+.page-list > th[data-name="page-name"] {
   width: 33%;
 }
 
-.page-list>th[data-name="url"] {
+.page-list > th[data-name="url"] {
   width: 40%;
 }

--- a/src/components/page-list/page-list.jsx
+++ b/src/components/page-list/page-list.jsx
@@ -75,7 +75,7 @@ export default class PageList extends React.Component {
     const onClick = this.didClickRow.bind(this, record);
 
     return (
-      <tr styleName="page-list" key={record.uuid} onClick={onClick}>
+      <tr key={record.uuid} onClick={onClick}>
         <td>{record.latest ? dateFormatter.format(record.latest.capture_time) : 'No saved versions'}</td>
         <td>{formatSites(record.tags)}</td>
         <td>{record.title}</td>

--- a/src/components/page-list/page-list.jsx
+++ b/src/components/page-list/page-list.jsx
@@ -47,7 +47,7 @@ export default class PageList extends React.Component {
     return (
       <div className="row">
         <div className="col-md-12">
-          <table className="page-list table">
+          <table styleName="page-list" className="table">
             <thead>
               {this.renderHeader()}
             </thead>

--- a/src/components/page-list/page-list.jsx
+++ b/src/components/page-list/page-list.jsx
@@ -1,8 +1,8 @@
-import {dateFormatter, formatSites} from '../scripts/formatters';
-import Loading from './loading';
+import {dateFormatter, formatSites} from '../../scripts/formatters';
+import Loading from '../loading';
 import React from 'react';
-import SearchBar from './search-bar/search-bar';
-
+import SearchBar from '../search-bar/search-bar';
+import './page-list.css';
 /**
  * These props also inherit from React Router's RouteComponent props
  * @typedef {Object} PageListProps

--- a/src/components/page-list/page-list.jsx
+++ b/src/components/page-list/page-list.jsx
@@ -47,7 +47,7 @@ export default class PageList extends React.Component {
     return (
       <div className="row">
         <div className="col-md-12">
-          <table styleName="page-list" className="table">
+          <table className="table" styleName="page-list" >
             <thead>
               {this.renderHeader()}
             </thead>
@@ -63,10 +63,10 @@ export default class PageList extends React.Component {
   renderHeader () {
     return (
       <tr>
-        <th styleName="page-list-th" data-name="capture-date">Capture Date</th>
-        <th styleName="page-list-th" data-name="site">Site</th>
-        <th styleName="page-list-th" data-name="page-name">Page Name</th>
-        <th styleName="page-list-th" data-name="url">URL</th>
+        <th data-name="capture-date">Capture Date</th>
+        <th data-name="site">Site</th>
+        <th data-name="page-name">Page Name</th>
+        <th data-name="url">URL</th>
       </tr>
     );
   }
@@ -75,11 +75,11 @@ export default class PageList extends React.Component {
     const onClick = this.didClickRow.bind(this, record);
 
     return (
-      <tr styleName="page-list-tbody-tr" key={record.uuid} onClick={onClick}>
-        <td styleName="page-list-td">{record.latest ? dateFormatter.format(record.latest.capture_time) : 'No saved versions'}</td>
-        <td styleName="page-list-td">{formatSites(record.tags)}</td>
-        <td styleName="page-list-td">{record.title}</td>
-        <td styleName="page-list-td"><a href={record.url} target="_blank" rel="noopener">{record.url}</a></td>
+      <tr styleName="page-list" key={record.uuid} onClick={onClick}>
+        <td>{record.latest ? dateFormatter.format(record.latest.capture_time) : 'No saved versions'}</td>
+        <td>{formatSites(record.tags)}</td>
+        <td>{record.title}</td>
+        <td><a href={record.url} target="_blank" rel="noopener">{record.url}</a></td>
       </tr>
     );
   }

--- a/src/components/page-list/page-list.jsx
+++ b/src/components/page-list/page-list.jsx
@@ -63,10 +63,10 @@ export default class PageList extends React.Component {
   renderHeader () {
     return (
       <tr>
-        <th data-name="capture-date">Capture Date</th>
-        <th data-name="site">Site</th>
-        <th data-name="page-name">Page Name</th>
-        <th data-name="url">URL</th>
+        <th styleName="page-list-th" data-name="capture-date">Capture Date</th>
+        <th styleName="page-list-th" data-name="site">Site</th>
+        <th styleName="page-list-th" data-name="page-name">Page Name</th>
+        <th styleName="page-list-th" data-name="url">URL</th>
       </tr>
     );
   }
@@ -75,11 +75,11 @@ export default class PageList extends React.Component {
     const onClick = this.didClickRow.bind(this, record);
 
     return (
-      <tr key={record.uuid} onClick={onClick}>
-        <td>{record.latest ? dateFormatter.format(record.latest.capture_time) : 'No saved versions'}</td>
-        <td>{formatSites(record.tags)}</td>
-        <td>{record.title}</td>
-        <td><a href={record.url} target="_blank" rel="noopener">{record.url}</a></td>
+      <tr styleName="page-list-tbody-tr" key={record.uuid} onClick={onClick}>
+        <td styleName="page-list-td">{record.latest ? dateFormatter.format(record.latest.capture_time) : 'No saved versions'}</td>
+        <td styleName="page-list-td">{formatSites(record.tags)}</td>
+        <td styleName="page-list-td">{record.title}</td>
+        <td styleName="page-list-td"><a href={record.url} target="_blank" rel="noopener">{record.url}</a></td>
       </tr>
     );
   }

--- a/src/components/web-monitoring-ui.jsx
+++ b/src/components/web-monitoring-ui.jsx
@@ -9,7 +9,7 @@ import Loading from './loading';
 import LoginForm from './login-form';
 import NavBar from './nav-bar';
 import PageDetails from './page-details';
-import PageList from './page-list';
+import PageList from './page-list/page-list';
 import VersionRedirect from './version-redirect';
 
 const configuration = window.webMonitoringConfig;

--- a/src/css/styles.css
+++ b/src/css/styles.css
@@ -126,40 +126,6 @@ body {
   padding: 0;
 }
 
-/* ===== List View ===== */
-.page-list {
-  table-layout: fixed;
-}
-
-.page-list tbody tr {
-  cursor: pointer;
-}
-
-.page-list tbody tr:hover {
-  background-color: #eee;
-}
-
-.page-list td {
-  overflow-wrap: break-word;
-  word-wrap: break-word;
-}
-
-.page-list th[data-name="capture-date"] {
-  width: 12%;
-}
-
-.page-list th[data-name="site"] {
-  width: 15%;
-}
-
-.page-list th[data-name="page-name"] {
-  width: 33%;
-}
-
-.page-list th[data-name="url"] {
-  width: 40%;
-}
-
 /* ===== Page Detail View ===== */
 .page-title {
   font-size: 1.5em;

